app/build.gradle

```
@@ -13,13 +13,13 @@ android {
             storePassword keystoreProperties['storePassword']
         }
     }
-    compileSdkVersion 31
+    compileSdkVersion 34
     defaultConfig {
-        applicationId "com.talkable.demo"
-        minSdkVersion 16
-        targetSdkVersion 31
-        versionCode 34
-        versionName "0.5.3"
+        applicationId "com.talkable.demoapp"
```

please change it to `appdemo` as I had to create a new release and used this name 
```suggestion
        applicationId "com.talkable.appdemo"
```